environment/deployments/roundtable/main.tf

```diff
@@ -267,6 +267,82 @@ resource "google_service_account" "reaper_sa" {
   project      = module.project_factory.project_id
 }
 
+# WorkloadIdentityPool for workloads to use Workload Identity Federation
+# instead of maintaining long-lived service account creds.
+resource "google_iam_workload_identity_pool" "pool" {
+  workload_identity_pool_id = "main"
+  display_name              = "Main"
+  description               = "Main identity pool for this project"
+  project                   = module.project_factory.project_id
+}
```

Comment:
I'm pretty sure we only need one pool (per project). We'd want multiple pools if we wanted different folks with different permissions for managing different providers, but that's not us.